### ISSUE TITLE
[IMPORTED FROM SCRATCH THREAD] - Godot Edition Not Working

### ISSUE THREAD

**Issue by AfterNetwork** [MikeDev101/catchatmmo#4]
Godot Edition is not working. Person that reported says that Godot Editiob doesn't work.
Operating system stats:
Using Windows NT 10.0 (Windows 10), Firefox 67.0 (Firefox OR Firefox 67), obviously not using flash, x64 version
Person that reported also said:
Whatever I try to open it, I will get a pop-up that said, “This app can't run on your PC - To find a version for your PC, check with the software publisher.”
In other words, it shows an error when trying to open it.
We have the link for the problem here:
https://scratch.mit.edu/discuss/topic/345604/?page=8#post-3558264

**Comment by MikeDev101** [comment]
MikeDEV here, I'm going to investigate the issue of why the windows version isn't working. I will update this thread accordingly on what I find.

MikeDEV
Creator and manager
of The CatChat MMO Project

**Comment by MikeDev101** [comment]
Hello again, I uploaded the x32 and x64 version of the Windows export. By default, the forum's download page defaults to downloading the x32 version, which should work on x64 windows.

If not, Let us know.

MikeDEV
Creator and manager
of The CatChat MMO Project
